-1

Tested at: d0b52458b494074c31cdf5804cacf1972f3d249f

You can see the results of the tests here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-17925/18428/summary.html

I found follow errors while testing this PR

**Failed tests:  Build**

* **Build**:

I found an error when building: 
 
 <pre>>> Compiling edm plugin /build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_1_X_2017-03-14-1600/src/RecoMuon/GlobalTrackingTools/plugins/Module.cc 
>> Compiling edm plugin /build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_1_X_2017-03-14-1600/src/RecoMuon/GlobalTrackingTools/plugins/GlobalTrackQualityProducer.cc 
Entering library rule at RecoMuon/L3MuonProducer
>> Compiling edm plugin /build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_1_X_2017-03-14-1600/src/RecoMuon/L3TrackFinder/plugins/SealModules.cc 
/build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_1_X_2017-03-14-1600/src/RecoMuon/L3TrackFinder/src/L3MuonTrajectoryBuilder.cc: In static member function 'static void L3MuonTrajectoryBuilder::fillDescriptions(edm::ParameterSetDescription&)':
/build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_1_X_2017-03-14-1600/src/RecoMuon/L3TrackFinder/src/L3MuonTrajectoryBuilder.cc:76:55: error: no matching function for call to 'MuonTrackingRegionBuilder::fillDescriptions(edm::ParameterSetDescription&)'
    MuonTrackingRegionBuilder::fillDescriptions(descTRB);
                                                       ^
In file included from /build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_1_X_2017-03-14-1600/src/RecoMuon/L3TrackFinder/src/L3MuonTrajectoryBuilder.cc:48:0:
/build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_1_X_2017-03-14-1600/src/RecoMuon/GlobalTrackingTools/interface/MuonTrackingRegionBuilder.h:63:17: note: candidate: static void MuonTrackingRegionBuilder::fillDescriptions(edm::ConfigurationDescriptions&)
     static void fillDescriptions(edm::ConfigurationDescriptions& descriptions);
</pre>

--- a/RecoMuon/L3TrackFinder/src/L3MuonTrajectoryBuilder.cc
+++ b/RecoMuon/L3TrackFinder/src/L3MuonTrajectoryBuilder.cc
@@ -73,7 +73,7 @@ L3MuonTrajectoryBuilder::~L3MuonTrajectoryBuilder() {
 
 void L3MuonTrajectoryBuilder::fillDescriptions(edm::ParameterSetDescription& desc) {
    edm::ParameterSetDescription descTRB;
-   MuonTrackingRegionBuilder::fillDescriptions(descTRB);
+   MuonTrackingRegionBuilder::fillDescriptionsHLT(descTRB);
    desc.add("MuonTrackingRegionBuilder",descTRB);
 }
 

